### DISCUSSION TITLE
Naming convention for libraries

### DISCUSSION THREAD

**Discussion by JasonMarechal25** [AntaresSimulatorTeam/Antares_Simulator#1606] [General]
Currently we have mix naming for libraries.
Libraries are typically named after there directory or existing naming. This lead to some inconsistency.

This lead to three questions:

- Which convention for directories ? (PascalCase, camelCase, snake_case, kebab-case, flatcase, Capitalised_Snake_Case, Capitalised-Kebab-Case)
- Which convention for libraries ?
- Should libraries and directories follow the same convention ?

**Comment by flomnes** [comment]
See also #1602

**Comment by flomnes** [comment]
I'd personally go with kebab-case for CMake targets & directories

antares-core

test-case-adequacy-patch

info-collector
